src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,7 +1,16 @@
 package mate.academy
 
 fun greetUsers() {
-    // implement function
+   var nameHere: Boolean = true
+    do {
+        println("Please enter a name (or press Enter to quit):")
+        val name: String? = readLine()
+        if (!name.isNullOrEmpty()) {
```

Comment:
You may simplify if-else via moving condition to while(). In this case nameHere variable would not be needed